core/arch/arm/kernel/sub.mk

```diff
@@ -1,13 +1,8 @@
 ifeq ($(CFG_WITH_USER_TA),y)
-srcs-y += user_ta.c
 srcs-y += ldelf_loader.c
```

Comment:
Please use `srcs-$(CFG_WITH_USER_TA) += ldelf_loader.c` instead of the `ifeq (... endif`